meta.go

```diff
@@ -39,7 +39,7 @@ var (
 var metaKeyValidator = regexp.MustCompile(`^(\w+([-:]*\w+)*)+(((\[\]|\[(0|[1-9]\d*)\]))?(\.(\w+([-:]*\w+)*)+)*)*$`)
 var rightBracketRegExp = regexp.MustCompile(`\[(.*?)\]`)
 var isNumberRegExp = regexp.MustCompile(`^[+-]?(?:[0-9]*[.])?[0-9]+$`)
-var metaKeyIsParameterRegExp = regexp.MustCompile(`^parameters\.(.+)`)
+var metaKeyIsParameterRegExp = regexp.MustCompile(`^parameters(:?\.(.+))?`)
```

Comment:
Should the regex be to correctly indicate the non-capturing group `^parameters(?:\.(.+))?` ?